source/User_Guide/Settings/teammates.md

```diff
@@ -153,33 +83,33 @@ You can find any access requests listed under **Pending Requests** on the **Team
 To grant that teammate access to the feature, click **Approve**.
 
 {% warning %}
-**By clicking "Approve", you are giving that teammate full access to the feature!** If you only want to give them limited access to the feature, you must navigate back to their teammate settings to make adjustments to their granular permissions.
+**By clicking "Approve" you are giving that teammate full access to the feature!** If you only want to give them limited access to the feature, you must navigate back to their teammate settings to make adjustments to their granular permissions.
 {% endwarning %}
 
-If you do not want to grant them access, simply click **Deny**.
-
-![Granting Access to a Feature]({{root_url}}/images/teammates_9.png)
+If you do not want to grant them access, click **Deny**.
 
 {% info %}
-A notification email will be sent to the teammate when they are either granted or denied access. This email will be delivered by SendGrid, but the "reply to" address will be the email address on file for the account owner.
+The teammate receives a notification email when they are either granted or denied access. SendGrid delivers these emails, but the "reply to" address is the email address on file for the account owner.
 {% endinfo %}
 
-![Access Granted Notification]({{root_url}}/images/teammates_11.png)
-
 {% anchor h2 %}
 Managing Teammates
 {% endanchor %}
 
-To modify an existing teammate's permissions, navigate to the **Teammates Page** under **Settings**. Under the list of current teammates, click the action menu next to the teammate you would like to edit.
+_To modify an existing teammate's permissions:_ 
 
-Click **Edit** to open a modal window presenting the teammate's current permissions. Make your desired changes and click **Update**.
+1. Navigate to the **Teammates Page** under **Settings**. 
+1. Under the list of current teammates, click the action menu next to the teammate you would like to edit.
+1. Click **Edit** to open a modal window presenting the teammate's current permissions. 
+1. Make your desired changes and click **Update**.
 
-To delete a teammate, navigate to the **Teammates Page** under **Settings**. Click the action menu next to the teammate you want to delete and click **Delete**.
+_To delete a teammate:_
 
-![Deleting a Teammate]({{root_url}}/images/teammates_10.png)
+1. Navigate to the **Teammates Page** under **Settings**. 
+1. Click the action menu next to the teammate you want to delete and click **Delete**.
 
 {% warning %}
-You cannot recover a deleted teammate! Please be certain before deleting one of your teammates.
+You cannot recover deleted teammates. Once a teammate has been deleted, it can never be restored. Any templates, campaigns, contacts, or API keys created by the teammate will not be deleted.
```

Comment:
Any templates, campaigns, contacts, or API keys created by the teammate will not be deleted by zombies.

Reply:
Braaaaaaiiiiiinssss